### ISSUE TITLE
malformed gres string

### ISSUE THREAD

**Issue by wccropper** [giovtorres/slurmtools#4]
Hi Giovanni,

I get this error when running show-cluster-util

Malformed gres string: gpu:k80:4(IDX:0-3)

Regards,
Chad

**Comment by giovtorres** [comment]
@wresch Did you ever write an updated gres parser to handle the newer gres string format?

**Comment by wresch** [comment]
yes our internal show-cluster-util handles it. but it does so with an older pyslurm. trying to test with a newer version now, but having some trouble getting pyslurm to build.

**Comment by wresch** [comment]
created pull request #5

**Comment by giovtorres** [comment]
Thanks @wresch !

@wccropper I merged PR #5.  Could you give it a try now?

**Comment by wccropper** [comment]
I no longer get the malformed GRES error. BUt I am on SLURM 18.08.3. Here is my test and output:
$ conda list | grep pyslurm
pyslurm                   18.8.3.1                  <pip>
$ conda list | grep hostlist
python-hostlist           1.17                      <pip>
$ ./show-cluster-util 
Traceback (most recent call last):
  File "./show-cluster-util", line 37, in <module>
    die("This tool requires pyslurm")
NameError: name 'die' is not defined

**Comment by wresch** [comment]
@wccropper - did you edit the path to pyslurm at

https://github.com/giovtorres/slurmtools/blob/0d27225ac297ff839000502a76d46f5fc706a9cb/show-cluster-util/show-cluster-util#L32

it's somewhat awkwardly included in the script rather than a config file...

**Comment by wccropper** [comment]
It gives the same error whether I use the local python 2.7.5 site-packages or the conda env site-packages paths. I have installed the 18.08.3 pyslurm on both.
